firmware_DavidBoard/Core/Src/TemperatureCalc.c

```diff
@@ -0,0 +1,94 @@
+/*
+ * TempCalc.c
+ *
+ *  Created on: Mar 13, 2022
+ *      Author: JacobS
+ */
+
+#include "TemperatureCalc.h"
+
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
+
+/***********************************************************************************************************************
+ * Variables
+ **********************************************************************************************************************/
+
+// HAL handle
+/*extern I2C_HandleTypeDef hi2c2;
+static uint8_t rawAdcData[16];
+static bool adcDataNew;
+static I2CBusStatus_t currentBusStatus;*/
+
+
+//static double V1;
+//static double R1;
+//static double T_K_inverse;
+static double TCelsius;
+static int V1000int;
+static int R1000int;
+
+
+/***********************************************************************************************************************
+ * Prototypes
+ **********************************************************************************************************************/
+
+int Temperature_ADCtoCelsius(ThermistorADC_t *ThermADCValues, DataBuffer_t *DataBuffer);
+
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
+
+<<<<<<< HEAD
+int Temperature_ADCtoCelsius(ThermistorADC_t *ThermADCValues, DataBuffer_t *DataBuffer) //change arguments - don't pass ThermADCValues
+=======
+int Temperature_ADCtoCelsius(ThermistorADC_t *ThermADCValues, DataBuffer_t *DataBuffer)
```

Comment:
This is the result of a merge conflict you didn't resolve. This obviously will not compile, Please fix.